### PR TITLE
#765 - Updating the troubleshooting.md

### LINKED ISSUE #692: multi line codes in <pre> turn to one line after mail send.
#### Issue Summary

I use mail to write my blog.
with markdown syntax.
```
int a = 10;
int b = 10;
int c = 10;
```
it will generate below html code:
<pre><code>int a = 10;
int b = 10;
int de = 10;
</code></pre>

then I send this html code to my blogger.
but in the post of my blog, the html change
<pre><code>int a = 10; int b = 10; int de = 10;</code></pre>
All codes are in one line, which is not my want.

Would you help me to work around for it please ?

#### Steps to Reproduce
          sgMail.setApiKey(process.env.SENDGRID_API_KEY);
          let msg = {
            to: 'errong.leng@blogger.com',
            from: 'errong.leng@gmail.com',
            subject: title,
            html: html,
          };
         sgMail.send(msg);

I confirm that html is correct and the codes are multi lines.
console.log(html)'s output is
<pre><code>int a = 10;
int b = 10;
int de = 10;
</code></pre>

I wish I can get multi lines code in my post also.
but in fact, all codes are placed in one line in my post.
<pre><code>int a = 10; int b = 10; int de = 10;</code></pre>

What should I do with html here ?

#### Technical details:

* sendgrid-nodejs Version: master (latest commit: [commit number])
* Node.js Version: 4.1.2

### PR DESCRIPTION
**Pull request by ssiddhantsharma**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

### Short description of what this PR does:
- **Html tags to support line wrap while writing the mail.**
 - Added the #692 multi-line codes in pre turn to one line after mail send, to the troubleshooting.md on your request.

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by ssiddhantsharma** [comment]
@thinkingserious Updated the troubleshoot file with your request.
https://github.com/sendgrid/sendgrid-nodejs/pull/785

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -126,3 +127,45 @@ const mail = Mail.create(data);
...
+You can also wrap the text in the HTML to make a multi-line blog post:
+```javascript
+<div style="white-space: pre-wrap;"> 
+<code>        int a = 10;
+              int b = 10;
+              int d = 10;
+</code>
+</div>
+```
+
```
Comment (reedsa) [inline TROUBLESHOOTING.md]: Please remove extraneous newlines from end of file.

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -126,3 +127,45 @@ const mail = Mail.create(data);
...
 console.log(body);
 ```
+
+<a name="wrapping-text"></a>
+## Wrapping Text
+
+You can write blog posts using e-mail with the help of SENDGRID API, like so:
+```javascript
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+      let msg = {
```
Comment (reedsa) [inline TROUBLESHOOTING.md]: Adjust to use proper indentation for these blocks.

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -126,3 +127,45 @@ const mail = Mail.create(data);
...
+        from: '<your-name>@gmail.com',
+        subject: title,
+        html: html,
+      };
+     sgMail.send(msg);
+``` 
+You can also wrap the text in the HTML to make a multi-line blog post:
+```javascript
+<div style="white-space: pre-wrap;"> 
+<code>        int a = 10;
```
Comment (reedsa) [inline TROUBLESHOOTING.md]: Newline after `<code>` tag and adjust block to use two-space indentation.

File: TROUBLESHOOTING.md [hunk 4]
```diff
@@ -126,3 +127,45 @@ const mail = Mail.create(data);
 const body = mail.toJSON();
 console.log(body);
 ```
+
+<a name="wrapping-text"></a>
+## Wrapping Text
+
+You can write blog posts using e-mail with the help of SENDGRID API, like so:
```
Comment (reedsa) [inline TROUBLESHOOTING.md]: Please use `SendGrid` rather than all caps.

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (ssiddhantsharma) [reply TROUBLESHOOTING.md]: @reedsa Made the changes, thank you for reviewing this one and helping me with good code practices!

File: TROUBLESHOOTING.md (hunk 2 above)
Comment (ssiddhantsharma) [reply TROUBLESHOOTING.md]: @reedsa Made the changes, thank you for reviewing this one and helping me with good code practices!

File: TROUBLESHOOTING.md (hunk 4 above)
Comment (ssiddhantsharma) [reply TROUBLESHOOTING.md]: @reedsa Made the changes, thank you for reviewing this one and helping me with good code practices!

File: TROUBLESHOOTING.md (hunk 3 above)
Comment (ssiddhantsharma) [reply TROUBLESHOOTING.md]: @reedsa Made the changes, thank you for reviewing this one and helping me with good code practices!

**Comment by ssiddhantsharma** [comment]
@reedsa Updated the file with required changes. Thank you for reviewing this one.

File: TROUBLESHOOTING.md (hunk 4 above)
Comment (reedsa) [inline TROUBLESHOOTING.md]: SendGrid still appears in all capital letters. Please format it as `SendGrid`

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (reedsa) [reply TROUBLESHOOTING.md]: I still see quite a few empty lines at the end of this file. Please remove them and I'll be happy to approve.

**Pull request merged**